integration

```diff
@@ -1076,6 +1076,7 @@
   "Pyhass/Hive-Custom-Component",
   "pypolestar/polestar_api",
   "QNimbus/haefele-connect-mesh",
+  "qqaatw/JciHitachiHA",
```

Comment:
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>

Reply:
yes.
> the **released** `hacs.json` file needs a country key